src/main/jte/auth/login.jte

```diff
@@ -26,24 +26,24 @@
                 <span>${context.localizeOrDefault(message, message)}</span>
             </div>
         @endif
-        <a class="input btn btn-primary" href="/oauth2/authorization/kit">${context.localize("login.kit")}</a>
+        <a class="input btn btn-primary" href="/oauth2/authorization/kit" aria-label="${context.localize("login.kit_login")}" title="${context.localize("login.kit_login")}">${context.localize("login.kit")}</a>
```

Comment:
We already have `login.kit`. A different name (`login.kit.description`?) would be better